README.md

```
@@ -15,7 +15,7 @@
 ![npm](https://img.shields.io/npm/v/@jsverse/transloco)
 ![Bundle Size](https://img.shields.io/bundlephobia/min/@jsverse/transloco)
 ![Downloads](https://img.shields.io/npm/dm/@jsverse/transloco)
-[![Build Status](https://github.com/jsverse/transloco/actions/workflows/ci.yml/badge.svg)]()
+[![Build Status](https://github.com/jsverse/transloco/actions/workflows/ci.yml/badge.svg)](<>)
```

markdownlint-cli2 (0.23.1)

[warning] No empty links

(MD042, no-empty-links)